Snowflake.Data.Tests/IntegrationTests/SFConnectionIT.cs

```diff
@@ -1790,6 +1835,11 @@ public void TestKeepAlive()
                 Assert.Fail();
             }
         }
+        
+        private string ResolveHostToUseFromTestConfig(string connectionString)
```

Comment:
connectionString is not used, such a method should be rather in the base class where testConfig is defined

Reply:
Moved to base class and removed unused parameter.

Reply:
now the builds failed because you are still passing the unneeded parameter to this method. You could also think about a shorter name like `ResolveHost()`

Reply:
you could skip `this` in `this.ResolveHost()`

Reply:
Done